File: editorial_ch/chap5-9/prob5-9-2.py
# 輸入
N = int(input())
A = list(map(int, input().split()))
B = list(map(int, input().split()))

# 將陣列 A, B 排序
A.sort()
B.sort()

# 求出答案
Answer = 0
for i in range(N):
	Answer += abs(A[i] - B[i])

# 輸出
print(Answer)

File: editorial_ch/chap5-9/prob5-9-2.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		// 輸入
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		long[] A = new long[N];
		long[] B = new long[N];
		for (int i = 1; i <= N; i++) A[i - 1] = sc.nextLong();
		for (int i = 1; i <= N; i++) B[i - 1] = sc.nextLong();
		
		// 將陣列 A, B 排序
		Arrays.sort(A);
		Arrays.sort(B);
		
		// 求出答案
		long Answer = 0;
		for (int i = 1; i <= N; i++) {
			Answer += Math.abs(A[i - 1] - B[i - 1]);
		}
		
		// 輸出
		System.out.println(Answer);
	}
}